src/timestamp.js

```diff
@@ -63,16 +63,16 @@ function calculateMessageReceivedTextFromDate(date, current=null) {
     // string to show
     let yearDelta = beginningOfYear.get_year() - datetime.get_year();
     if (yearDelta === 1) {
-        return 'Last year';
+        return _('Last year');
     } else if (yearDelta > 0) {
-        return ['About', yearDelta, 'years ago'].join(' ');
+        return [_('About'), yearDelta, _('years ago')].join(' ');
```

Comment:
This one should be `_("About %d years ago").format(yearDelta)` (same below for months, weeks and days)